weblogic/tests/compose/weblogic/Dockerfile

```diff
@@ -0,0 +1,93 @@
+#Copyright (c) 2014, 2020, Oracle and/or its affiliates.
```

Comment:
Why not just pull the image that has the Dockerfile logic already defined (along with scripts) and then extend that? That way you probably wouldn't need to also add in the shell/python scripts the original Dockerfile needed to run. 

I had to do something similar for Oracle Database with modifying the Dockerfile, but found it was a lot easier just to use the completed image instead of the base image:

My Dockerfile: https://github.com/DataDog/integrations-core/pull/10591/files#diff-73ee12464f1d0c7dffb438fa813e8f7bddd657d87d9ba8aa9c115893438e1505R3

Original image Dockerfile: https://github.com/oracle/docker-images/blob/main/OracleDatabase/SingleInstance/dockerfiles/12.2.0.1/Dockerfile

Reply:
Good point, I'll give that a try

Reply:
The Dockerfile does pull the Oracle WebLogic image. The additional scripts create a domain and deploy a sample application to replicate a WebLogic cluster setup. To reduce bloat, I've removed most of the generic scripts and download them as part of the Dockerfile image build steps.